### PR TITLE
FEAT: Added into keyboard_layout file more layouts.

### PR DESCRIPTION
**Pull request by isaacsa51**
Implemented more keyboard layouts into the `keyboard_layout.dart` file, including layouts as:

 - Gallium V2
 - Sturdy
 - Canaria

Still hope and don't be afraid to reach if anything isn't working or any help with the split/matrix layout view.

### CODE REVIEW

**Comment by conventoangelo** [comment]
I will test it on my machine when I get home. Thanks for your help!

File: lib/keyboard_layouts.dart [hunk 1]
```diff
@@ -115,6 +115,36 @@ const graphite = KeyboardLayout(
...
+    ['N', 'R', 'T', 'S', 'G', 'Y', 'H', 'A', 'E', 'I', ','],
+    ['Q', 'X', 'M', 'C', 'V', 'K', 'P', '.', '-', '/'],
+    [' '],
+  ],
+);
+
+const sturdy = KeyboardLayout(
+  name: 'Sturdy',
+  keys: [
+    ['V', 'M', 'L', 'C', 'P', "X", 'F', 'O', 'U', '-', '[', ']'],
```
Comment (conventoangelo) [inline lib/keyboard_layouts.dart:131]: Replace '-' with J.

File: lib/keyboard_layouts.dart [hunk 2]
```diff
@@ -115,6 +115,36 @@ const graphite = KeyboardLayout(
...
+    ['S', 'T', 'R', 'D', 'Y', '.', 'N', 'A', 'E', 'I', '/'],
+    ['Z', 'K', 'Q', 'G', 'W', 'B', 'H', "'", ';', ','],
```
Comment (conventoangelo) [inline lib/keyboard_layouts.dart:133]: ZKQGW ordering is for split keyboards. Change into Sturdy ANSI with Angle Mod, i.e., KQGWZ. (This is just based on MonkeyType as I'm not familiar with Sturdy layout.

File: lib/keyboard_layouts.dart [hunk 3]
```diff
@@ -115,6 +115,36 @@ const graphite = KeyboardLayout(
   ],
 );
 
+const galliumV2 = KeyboardLayout(
+  name: 'Gallium V2',
+  keys: [
```
Comment (conventoangelo) [inline lib/keyboard_layouts.dart:120]: Many keys misplaced. Refer to the [Gallium repo](https://github.com/GalileoBlues/Gallium) on implementation of ANSI Gallium V2/Gallium_rowstag. Attached is a picture of MonkeyType's implementation of Gallium V2.
image

**Review by conventoangelo — CHANGES_REQUESTED** [review]
Hey Isaac,

Thank you so much for this awesome contribution! 🎉 It's great to see someone jumping in and helping out with the project.

I tested things out, and while the changes look promising, there are a few tweaks needed with the key placements before it's ready for merge. Once those are addressed, I'll be more than happy to approve and merge it right away!

For now, let's stick with the ANSI layout. I'll brainstorm some ideas for switching between the split layout and ANSI, given the format of the keyboard layouts in the 'keyboard_layouts.dart' file.

Really appreciate your enthusiasm, and a **_special thank you_** for being the first contributor to the project—this means a lot! 🙌

File: lib/keyboard_layouts.dart (hunk 1 above)
Comment (isaacsa51) [reply lib/keyboard_layouts.dart]: You can check [ba0a267](https://github.com/conventoangelo/OverKeys/pull/1/commits/ba0a26751d1895b6e6665741de7b719c2e325efe), it's resolved.

Sorry about that, I used [oxey.dev](https://oxey.dev/playground/index.html) to see the layouts.

**Comment by isaacsa51** [comment]
All comments resolved on: [ba0a267](https://github.com/conventoangelo/OverKeys/pull/1/commits/ba0a26751d1895b6e6665741de7b719c2e325efe)

**Comment by isaacsa51** [comment]
Added one more layout too: [Hands down Neu](https://sites.google.com/alanreiser.com/handsdown)

**Review by conventoangelo — APPROVED** [review]
Looks good to me! 🚀 Thank you again for making this PR.

**Pull request merged**